runtime/interp.c

```diff
@@ -125,8 +134,8 @@ sp is a local copy of the global variable Caml_state->extern_sp. */
 #endif
 
 #define Check_trap_barrier \
-  if (Caml_state->trapsp >= Caml_state->trap_barrier) \
-    caml_debugger(TRAP_BARRIER, Val_unit)
+  if (domain_state->trap_sp_off >= domain_state->trap_barrier_off) \
+      caml_debugger(TRAP_BARRIER, Val_unit)
```

Comment:
Fix indentation.

Reply:
Fixed in https://github.com/ocaml-multicore/ocaml/commit/eebbb5ab5a3be66817892b032949ca9387e4df4f